### PR TITLE
Style 404

### PR DESCRIPTION
**Pull request by JoelLindow**
Related Issue: #121 

Summary: Styled

Author(s): Joel

### CODE REVIEW

File: public/404.html.erb [hunk 1]
```diff
@@ -58,10 +58,12 @@
...
   <div class="dialog">
     <div>
+      <img src="http://www.reactiongifs.com/wp-content/uploads/2013/08/no.gif" alt="">
       <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <p>Sometimes we rock out too hard and end up inwierd places too. That's when you have to take a moment and re-evaluate your life. Go back to a place you're familiar with and start over. So, get out of here. There's nothing to see here.</p>
+      <p><%= link_to "Back To Main Page", root_path %></p>
     </div>
-    <p>If you are the application owner check the logs for more information.</p>
+    <p>If you are the application owner check the logs for more information. If you're not, go back to the main page and buy something.</p>
```
Comment (limsammy) [inline public/404.html.erb:66]: Love it

**Review by limsammy — APPROVED** [review]
Looks good

File: public/404.html.erb [hunk 2]
```diff
@@ -58,10 +58,12 @@
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
+      <img src="http://www.reactiongifs.com/wp-content/uploads/2013/08/no.gif" alt="">
       <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <p>Sometimes we rock out too hard and end up inwierd places too. That's when you have to take a moment and re-evaluate your life. Go back to a place you're familiar with and start over. So, get out of here. There's nothing to see here.</p>
```
Comment (amhursh) [inline public/404.html.erb:63]: classy, effective

File: public/404.html.erb [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
   .rails-default-error-page div.dialog {
     width: 95%;
-    max-width: 33em;
+    max-width: 60em;
```
Comment (randy-darsh) [inline public/404.html.erb:17]: i appreciate the fact that you changed this to a 60em rather than a 33em

File: public/404.html.erb [hunk 4]
```diff
@@ -27,7 +27,7 @@
     border-top-left-radius: 9px;
     border-top-right-radius: 9px;
     background-color: white;
-    padding: 7px 12% 0;
+    padding: 7px 5% 0;
```
Comment (randy-darsh) [inline public/404.html.erb:30]: i also appreciate the fact that you changed the padding to 5% instead of 12%

**Comment by randy-darsh** [comment]
MERGING... ENGAGE!

**Pull request merged**